models/generated/directory/directory.go

```diff
@@ -295,3 +296,17 @@ type Gateway struct {
 	DnsNameserver  []string       `bson:"dns_nameserver" json:"dns_nameserver"`
 	FreeToUse      bool           `bson:"free_to_use" json:"free_to_use"`
 }
+
+type FarmerIP struct {
```

Comment:
`FarmerIP` is misleading. It can be changed to just `PublicIP`. It won't conflict with the reservation type 'PublicIP' because they are both live in a different package